src/scenes/level.js

```diff
@@ -275,4 +237,112 @@ export class LevelScene extends Phaser.Scene {
         this.stats.updateAutoClickerDPS(autoClicker.dps);
         this.characterData.numberOfAutoClickers++;
     }
+
+    clickAnimation() {
+        // Animation settings
+        let image = {},
+            imageName = "",
+            startX = 0,
+            startY = 0,
+            scale = 1,
+            curve = 0,
+            alpha = 1;
+
+        // Get current weapon image
+        let curWeapon = this.dashboard.equipment.obj.equipment.WEAPON;
+        if (Object.keys(curWeapon).length) {
+            // Set animation based on current weapon
+            switch (curWeapon.skill) {
+                // Melee weapons use the weapon model image
```

Comment:
I'm all for explanations in the comments but try to be wary of commenting something that should be self-evident from reading the code

Reply:
Removed that one comment

Reply:
Happens a good bit elsewhere but I don't care enough to force change. Just something to be cognizant of

Reply:
Alright